Airways/RNAV/M17.txt

```diff
@@ -1,7 +1,7 @@
-BEKDA BEKDA EXMOR EXMOR
+ADKIK ADKIK EXMOR EXMOR
 EXMOR EXMOR KESUP KESUP
 KESUP KESUP SWANY SWANY
 SWANY SWANY STU STU
 STU STU PEMOB PEMOB
-PEMOB PEMOB LEMGU LEMGU
-LEMGU LEMGU VATRY VATRY
\ No newline at end of file
+PEMOB PEMOB RIVOT RIVOT
```

Comment:
The original routing is fine (note in the document it jumps from page 62 to page 65), so the final section LEMGU VATRY is in page 63, not shown as it's not changing)

Reply:
You mean like this? 
248666c